src/main/java/org/myrobotlab/service/WebkitSpeechSynthesis.java

```diff
@@ -0,0 +1,75 @@
+package org.myrobotlab.service;
+
+import java.io.IOException;
+
+import org.myrobotlab.framework.ServiceType;
+import org.myrobotlab.logging.LoggerFactory;
+import org.myrobotlab.service.abstracts.AbstractSpeechSynthesis;
+import org.myrobotlab.service.data.AudioData;
+import org.slf4j.Logger;
+
+import com.amazonaws.services.polly.model.DescribeVoicesRequest;
+
+/**
+ * Amazon's cloud speech service
+ * 
+ * Free Tier The Amazon Polly free tier includes 5 million characters per month,
+ * for the first 12 months, starting from the first request for speech.
+ * 
+ * Polly Pricing Pay-as-you-go $4.00 per 1 million characters (when outside the
+ * free tier).
+ *
+ * @author GroG
+ *
+ */
+public class WebkitSpeechSynthesis extends AbstractSpeechSynthesis {

```

Comment:
This class is not named correctly.  There is no such thing as WebkitSpeechSynthesis.  Webkit is a speech recognizer.  I think perhaps you mean this to be Polly?  Or perhaps this class was added by accident?  Please remove/revert.

Reply:
No, its not incorrectly named.   It's "discovered".  In reading the documentation regarding webkitSpeechRecognizer, it turns out Chrome support SpeechSynthesis.  They are two independent functions so, they deserve to be separate.   I did some testing initially in WebkitSpeechRecognizer, but didn't want what I had learned to be there so I created the appropriate files.  This is on the same level as Polly, but no need of a key.